x-pack/plugins/triggers_actions_ui/public/types.ts

```diff
@@ -460,9 +467,19 @@ export interface FetchAlertData {
   onSortChange: (sort: EuiDataGridSorting['columns']) => void;
   refresh: () => void;
   sort: SortCombinations[];
-}
-
-export interface AlertsTableProps {
+  /**
+   * We need to have it because of lot code is expecting this format
+   * @deprecated
```

Comment:
could you please create an issue to remove this deprecated fields as we don't want to keep this here for ever?